namespaces/live.cloud-platform.service.justice.gov.uk/laa-civil-case-api-production/resources/ecr.tf

```diff
@@ -0,0 +1,25 @@
+/*
+ * Make sure that you use the latest version of the module by changing the
+ * `ref=` value in the `source` attribute to the latest version listed on the
+ * releases page of this repository.
+ *
+ */
+module "ecr" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=7.0.0"
+
+  # Repository configuration
+  repo_name = var.namespace
+
+  # OpenID Connect configuration
+  oidc_providers      = ["github"]
+  github_repositories = ["example-repository"]
```

Comment:
Is this an actual repo? The module will try to create secrets there so please point this value to a public repo.

Reply:
Update to laa-civil-case-api